draft-irtf-cfrg-opaque.md

```diff
@@ -1878,6 +1895,14 @@ implementation considerations.
   of the derivation of OPRF keys via a single PRF. As long as the derivation
   of different OPRF keys from a single OPRF has different PRF inputs, the
   protocol is secure. The choice of such inputs is up to the application.
+- {{JKX18}} comments on a strong defense against essentially the only
+  inherent remaining weakness of a "strong aPAKE", namely, offline
+  dictionary attacks upon server compromise or honest-but-curious servers.
+  The authors suggest implementing the OPRF phase as a Threshold OPRF {{TOPPSS}},
+  effectively forcing an attacker to act online or to control at least t key
+  shares, where t is the threshold number of shares necessary to recombine
+  the secret OPRF key. This implementation only affects the server and changes
```

Comment:
You should say explicitly that reconstructing the key does not represent a break of the system but only the ability to run offline dictionary attacks. That is, to be able to impersonate a user, an attacker needs to break into at least t OPRF servers and then run a dictionary attack against the specific user's password. Even breaking into all servers, still requires an exhaustive offline dictionary attack against the user's password. 
(Note to selves: It is a VERY STRONG protection.)